crates/python_utilities/src/ast_printer.rs

```diff
@@ -53,6 +55,14 @@ impl<'a> WritingBuffer<'a> {
     pub fn finish(self) -> String {
         self.buf.join("")
     }
+
+    pub fn begin_keyword(&mut self) {
+        self.in_keyword = true
+    }
+
+    pub fn finish_keyword(&mut self) {
```

Comment:
what about using return`&mut Self` chaining pattern return here?

Reply:
Addressed in 64150be9f93ffc954a02cfebda6f65460d294c17